python/extensions/pip.bzl

```
@@ -316,9 +437,55 @@ Each `pip.parse()` call configures a particular Python version. Multiple calls
 can be made to configure different Python versions, and will be grouped by
 the `hub_name` argument. This allows the same logical name, e.g. `@pip//numpy`
 to automatically resolve to different, Python version-specific, libraries.
+
+pip.whl_mods:
+This tag class is used to create different wheel modification JSON files.  These files
+contain directives that are used by the wheel_installer.py during the creation of 
+wheels.
 """,
     implementation = _pip_impl,
     tag_classes = {
-        "parse": tag_class(attrs = _pip_parse_ext_attrs()),
+        "parse": tag_class(
+            attrs = _pip_parse_ext_attrs(),
+            doc = """\
+This tag class is used to create a pip hub and all of the spokes that are part of that hub.
+This tag class reuses the pip attributes that are found in 
+@rules_python//python/pip_install:pip_repository.bzl
+""",
+        ),
+        "whl_mods": tag_class(
+            attrs = _mod_tag_attrs(),
+            doc = """\
+This tag class is used to create JSON file that are used when calling wheel_builder.py.  These
+JSON files contain instructions on how to modify a wheel's project.  Each of the attributes
+create different modifications based on the type of attribute. Previously to bzlmod these
+JSON files where referred to as annotations, and were renamed to whl_modifications in this
+extension.
```

Similar as other edits: remove low-level details not relevant to the user.

```suggestion
This tag class helps create JSON files to describe modifications to the BUILD files
generated for wheels. Each of the attributes create different modifications based on
the type of attribute. (In workspace-style builds, these were previously called
"annotations" or "package annotations")
```